CHANGELOG.md

```diff
@@ -1,6 +1,15 @@
 Scalyr Agent 2 Changes By Release
 =================================
 
+## 2.1.24 "Xuntian" - Oct 1, 2021
+
+<!---
+Packaged by Yan Shnayder <yans@sentinelone.com> on Oct 1, 2021 14:10 -0800
+--->
+
+Improvements:
+* Implemented optional line merging when running in Docker based Kubernetes to work around Docker's own 16KB line length limit. Use the ``merge_json_parsed_lines`` config option to enable this functionality.
```

Comment:
Can you also list the environment variable you need to set?  I believe most config options will have an env var by default.  It is better for folks on K8s / Docker.

Reply:
Or, is this a per-logfile config option?

Reply:
It is not.